spec/unit/pull_request_spec.rb

```diff
@@ -8,7 +8,58 @@
     end
   end
 
-  describe '.to_commit' do
+  describe '#story_ids' do
+    let(:story_id) { random_story_id }
+
+    subject { PullRequest.new(title: title).story_ids }
+
+    context 'no story ID' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.